src/main/resources/io/jenkins/plugins/pipelinegraphview/multipipelinegraphview/MultiPipelineGraphViewAction/sidepanel.jelly

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:p="/lib/hudson/project">
+    <l:side-panel>
+        <l:tasks>
+            <j:set var="buildUrl" value="${h.decompose(request)}"/>
+            <l:task icon="images/24x24/up.png" href="${buildUrl.baseUrl}/" title="${%Back to Build}" contextMenu="false"/>
```

Comment:
`${buildUrl.baseUrl}` here seems to be empty in practice. Filed <https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/55>.